ledger-core/v2/virtual/handlers/vstateunavailable.go

```diff
@@ -61,6 +61,11 @@ type stateAlreadyExistsErrorMsg struct {
 	GotState  string
 }
 
+type stateIsNotReadyErrorMsg struct {
```

Comment:
this should be defined in file where it is used

Reply:
@sexyhamster Fixed